### PR TITLE
[RFR][NOTEST]Adding/updating VMware manual tests

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================
__Adding tests__ new VMware Manual Tests.

### CODE REVIEW

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: **Required** we want this this to be linted .

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+"""Manual VMware Provider tests"""
+import pytest
+
+pytestmark = []
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: *Required** You can remove this empty marker

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    """
+    N-3 filters for esx provider.
+    Example: ESXi 6.5 is the current new release.
+    So filters for 6.5 (n), 6.0 (n-1), 5.5 (n-2) at minimum.
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Provisioning
+        caseimportance: low
+        initialEstimate: 1/4h
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: **Required** Need `testSteps` and `expectedResults` here and the below tests that are missing them. I would also like to see the doc blocks clearer on what is being tested here.

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    https://bugzilla.redhat.com/show_bug.cgi?id=1588908
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: critical
+        initialEstimate: 1/3h
+        testtype: integration
+        title: Test if VMs are marked as linked_clones(ideally
+        they shouldn't mark them all as linked_clone=t)
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: **Optional** The text within the parenthesis should be in the doc block.

File: cfme/tests/infrastructure/test_vmware_provider.py (hunk 3 above)
Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: Is it required for everything?

Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: Also does docblock translates to anything in polarion or just remains here in code?

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: Line 92 is causing travis to fail

Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: Fixed

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+    to vds port groups.
+    Compute > Infrastructure > Host > [Select host] > Properties > Network
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/12h
+        testtype: integration
+    testSteps:
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: The formatting is off here

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    to vds port groups. Check to see if you can navigate to DSwitch and tag it.
+    Compute > Infrastructure > Host > [Select host] > Properties > Network
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/12h
+        testtype: integration
+    testSteps:
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_vmware_provider.py]: formatting

Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: fixed.

File: cfme/tests/infrastructure/test_vmware_provider.py (hunk 6 above)
Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: fixed.

**Pull request merged**